src/navigation/AuthNavigator.tsx

```diff
@@ -0,0 +1,16 @@
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import SignUp from '@src/screens/signUp/SignUp';
+import Login from '@src/screens/login/login';
```

Comment:
Maintain proper blank lines with the correct order of import statements-
import {createNativeStackNavigator} from '@react-navigation/native-stack';

import SignUp from '@src/screens/signUp/SignUp';
import Login from '@src/screens/login/login';